server/channels/app/import_functions.go

```diff
@@ -1447,12 +1463,28 @@ func (a *App) importReplies(rctx request.CTX, data []imports.ReplyImportData, po
 				return model.NewAppError("importReplies", "app.post.save.app_error", nil, "", http.StatusInternalServerError).Wrap(err)
 			}
 		}
+		for _, created := range postsCreated {
+			reactions, ok := interimReactionsMap[created.CreateAt]
+			if !ok || reactions == nil {
```

Comment:
Would reactions ever be nil if `ok == true` ?

Reply:
It shouldn't but this is just a defensive programming :p